### DISCUSSION TITLE
What counts as a connection exactly in TipTap?

### DISCUSSION THREAD

**Discussion by AndyLiang1** [ueberdosis/tiptap#5441] [General Questions] [answered]
Hello all! I've been working on a personal project with TipTap. I have a TipTap cloud app set up. I have an editor that allows collaboration. I send a JWT from the backend using my app secret in dashboard settings and all was good. I was able to collab using multiple browsers (even tested using a friend).  

On my TipTap cloud dashboard, it says I have 0 connections and 0 max concurrent connections. I am confused because I have been testing my application so many times this week and as far as I know, when there is collaboration, you have to connect with web sockets. I have even checked my logs and they say 

User authenticated 03.08.2024 21:44:24
New connection to "test_doc" etc. 

I am unsure of how to create these "connections" they speak of and they just closed my app due to inactivity (although I manually resumed it). Any help would be super appreciated!

**Comment by timoisik** [comment, marked as answer]
Hey @AndyLiang1, 
we do not count the connections which are comming from localhost (your machine). Only your production connections will be counted then.

**Reply by AndyLiang1** [reply]
Thank you so much for the help on discord too! :)
